pyrs/core/reduce_hb2b_pyrs.py

```diff
@@ -674,6 +674,9 @@ def histogram_by_numpy(pixel_2theta_array, pixel_count_array, two_theta_bins, is
         var, var_edges = np.histogram(pixel_2theta_array, bins=two_theta_bins, weights=pixel_var_array**2)
         var = np.sqrt(var)
 
+        # define mask to set edge of histogram to zero with var 1
```

Comment:
The comment doesn't appear to agree with the code

Reply:
I updated the code comments in the new commit. I also changed the variable names for the histogram data and variance arrays to align the variable names with the data they represent